.github/workflows/codeql-analysis.yml

```
@@ -0,0 +1,112 @@
+---
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+name: CodeQL
+
+on:
+  merge_group:
+    types:
+      - checks_requested
+  push:
+    # Dependabot triggered push events have read-only access, but uploading code
+    # scanning requires write access.
+    branches-ignore:
+      - dependabot/**
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches:
+      - develop
```

Corrected sorting of keys. The updated comment reflects the current comment in [cisagov/skeleton-docker](https://github.com/cisagov/skeleton-docker/blob/7320dadd1527eed465e04ccb25d6c6bb105799d7/.github/workflows/codeql-analysis.yml#L14).

```suggestion
  pull_request:
    # The branches here must be a subset of the ones in the push key
    branches:
      - develop
  push:
    # Dependabot triggered push events have read-only access, but uploading code
    # scanning requires write access.
    branches-ignore:
      - dependabot/**
```